pattoo/db/schemas.py

```diff
@@ -95,7 +106,17 @@ def connection_resolver(
             self, resolver, connection, model, root, info, **args):
         query = resolver(
             root, info, **args) or self.get_query(model, info, **args)
+
+        if type(query) == AuthInfoField:
+            message = query.message
+
+            if query.message == "Invalid header padding":
+                message = "Invalid Token Provided"
+
+            raise GraphQLError(message)
```

Comment:
Also use `pattoo_shared.log2die` or `pattoo_shared.log2debug`, that way the message appears in the logs. This won't allow proper troubleshooting.

Reply:
@Dev-Dominic